pkg/apis/cluster/v1alpha1/well_known_taints.go

```diff
@@ -2,6 +2,14 @@ package v1alpha1
 
 const (
 	// TaintClusterUnscheduler will be added when cluster becomes unschedulable
-	// and removed when node becomes scheduable.
+	// and removed when cluster becomes scheduable.
 	TaintClusterUnscheduler = "cluster.karmada.io/unschedulable"
+
+	// TaintClusterNotReady will be added when cluster is not ready
+	// and removed when cluster becomes ready.
+	TaintClusterNotReady = "cluster.karmada.io/not-ready"
+	// TaintClusterUnreachable will be added when cluster becomes unreachable
+	// (corresponding to ClusterConditionReady status ConditionUnknown)
+	// and removed when cluster becomes reachable (ClusterConditionReady status ConditionTrue).
+	TaintClusterUnreachable = "cluster.karmada.io/unreachable"
```

Comment:
I can't well tell the differences between the two taints. Need some time to look into the changes made to the controller.

Reply:
`k/k` also does. The two taints describe different kinds of taint. They have effects on different tolerations, e.g., whether a workload could tolerate not ready clusters or unknown clusters.

Reply:
Yeah I think they are different